tests/package.json

```
@@ -0,0 +1,14 @@
+{
+    "scripts": {
+        "test:e2e": "cucumber-js --require cucumber.conf.js --require e2e/stepDefinitions/**/*.js --format @cucumber/pretty-formatter"
+    },
+    "devDependencies": {
+        "@cucumber/cucumber": "^10.3.2",
+        "@cucumber/pretty-formatter": "^1.0.1",
+        "@playwright/test": "^1.42.1",
+        "playwright": "^1.42.1"
```

you can only use `@playwright/test`
```suggestion
        "@playwright/test": "^1.44.0"
```